src/hotspot/share/classfile/javaClasses.cpp

```diff
@@ -2104,11 +2104,11 @@ void java_lang_VirtualThread::set_notified(oop vthread, jboolean value) {
 }
 
 jlong java_lang_VirtualThread::waitTimeout(oop vthread) {
-  return vthread->long_field(_waitTimeout_offset);
+  return vthread->long_field(_timeout_offset);
```

Comment:
Not sure what motivated the name change but it seems odd to have the method named differently to the field it accesses. ??

Reply:
It was initially parkTimeout and waitTimeout but it doesn't require two fields as you can't be waiting in Object.wait(timeout) and LockSupport.parkNanos at the same time. So the field was renamed, the accessors here should probably be renamed too.

Reply:
Renamed accessors.